README.md

```diff
@@ -24,4 +24,5 @@ You are given an array of goods. Render them in a table with the ability to sele
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_goods-selector/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO
+  LINK](https://ysrckr.github.io/react_goods-selector/) and add it to the PR description.
```

Comment:
The links are wrong in the hook tasks. Your demo points to the task with classes
```suggestion
  LINK](https://ysrckr.github.io/react_goods-selector-hooks/) and add it to the PR description.
```